### PR TITLE
[TGBOT-174] Create Book class

### PR DESCRIPTION
**Pull request by MarinaSauchuk**
TGBOT-174 Create Book class
https://i18nteam.atlassian.net/browse/TGBOT-174

### CODE REVIEW

**Review by lamaconda — APPROVED** [review]
lgfm

File: src/main/java/com/example/demo/model/Book.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
+package com.example.demo.model;
+
+import java.util.Objects;
+
+public class Book {
+    String title;
+    String author;
+    String language;
+    int pages;
+    int yearPublished;
```
Comment (lamaconda) [inline src/main/java/com/example/demo/model/Book.java]: Would be good if they become private

File: src/main/java/com/example/demo/model/Book.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
+package com.example.demo.model;
+
+import java.util.Objects;
+
+public class Book {
+    String title;
+    String author;
```
Comment (travka677) [inline src/main/java/com/example/demo/model/Book.java]: May be also replace variable author of type String  with variable of type Author

**Review by travka677 — COMMENTED** [review]
Variable author

File: src/main/java/com/example/demo/model/Book.java (hunk 2 above)
Comment (lamaconda) [reply src/main/java/com/example/demo/model/Book.java]: I think its redundant for now, we can add task in backlog about creating Author class

Comment (travka677) [reply src/main/java/com/example/demo/model/Book.java]: We already have such a task

File: src/main/java/com/example/demo/model/Book.java (hunk 1 above)
Comment (MarinaSauchuk) [reply src/main/java/com/example/demo/model/Book.java]: Right, thank you.

File: src/main/java/com/example/demo/model/Book.java (hunk 2 above)
Comment (lamaconda) [reply src/main/java/com/example/demo/model/Book.java]: > We already have such a task

my bad

**Review by max-java — APPROVED** [review]
lgfm

**Review by travka677 — APPROVED** [review]

**Review by lamaconda — APPROVED** [review]
lgfm

**Review by QewBye48 — APPROVED** [review]

**Review by QewBye48 — APPROVED** [review]

File: src/main/java/com/example/demo/model/Book.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
+package com.example.demo.model;
+
+import java.util.Objects;
+
+public class Book {
+    private String title;
+    private String author;
+    private String language;
+    private int pages;
+    private int yearPublished;
```
Comment (Grideron) [inline src/main/java/com/example/demo/model/Book.java:10]: add "private String Genre"

**Review by Grideron — APPROVED** [review]

**Pull request merged**